lib/go/test/go.mod

```diff
@@ -8,7 +8,6 @@ require (
 	github.com/onflow/flow-core-contracts/lib/go/templates v0.11.2-0.20220422202806-92ad02a996cc
 	github.com/onflow/flow-emulator v0.31.2-0.20220513151845-ef7513cb1cd0
 	github.com/onflow/flow-ft/lib/go/templates v0.2.0
-	github.com/onflow/flow-go v0.25.13-0.20220513151142-7858f76e703b
```

Comment:
was this the only reference to flow-go?

Reply:
- `flow-go` was imported to use domain separation tags from the protocol. Once the protocol is updated, I'm going to import the tags again, though this needs to be done over separate steps.
`github.com/onflow/flow-go/crypto` is also imported below but it that's a separate module.